apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/InpatientClaimTransformerV2.java

```diff
@@ -262,7 +259,8 @@ private static ExplanationOfBenefit transformClaim(
         claimGroup.getFiscalIntermediaryNumber(),
         claimGroup.getLastUpdated(),
         claimGroup.getFiDocumentClaimControlNumber(),
-        claimGroup.getFiscalIntermediaryClaimProcessDate());
+        claimGroup.getFiscalIntermediaryClaimProcessDate(),
+        Optional.of(claimGroup.getClaimQueryCode()));
```

Comment:
Previously the claim query code was not an optional when we mapped it above; is there an issue here where we might be allowing for an optional field here where it was not previously? Should we be ensuring the field exists if it should not be optional? (True for each of the transformers where we had claim query code, just picking this one as an example)

Reply:
so for all claim types that are not HHA and hospice it is mandatory. This is validated when the data comes in from CCW bc it is required.